### DISCUSSION TITLE
Possible to configure per notification rules for sms/email

### DISCUSSION THREAD

**Discussion by TD-4242** [magicbell/community#122] [Questions] [answered]
Is it possible to have rules on a per-notification within a category to send to browser/email/sms?

Within our application we have notification categories for real-time alerts and daily scans.  Customers may have 10-50+ of each and some they may want in app, some email and some sms.  I can only seem to turn it all on or all off for a category.  is there any way to set it so only certain notifications within a category are sent to each channel?

**Comment by TD-4242** [comment, marked as answer]
After some research I think this may have an example of what I'm looking for.
https://www.magicbell.com/docs/subscriptions

I think I can have a category of `scans` and a topic of `scan-<scan_id>` then have the user able to subscribe to the full category of scans or the individual topic of `scan-<scan_id>`

**Reply by unamashana** [reply]
Thanks for the answer - to add to it, topic subscriptions can be across categories, or more granularly at a category level. So yes, that should work well for you.
